src/digitalmarketplace/components/cookie-settings/README.md

```diff
@@ -0,0 +1,26 @@
+# Cookie Settings page
+
+This component includes Javascript only, for use the User Frontend Cookie Settings page.
+
+## Functionality
+On page load:
+- adds an event listener to the Cookie Settings form to handle any submit actions
+- populates the Cookie Settings form with the user's existing preferences (if set)
+- displays a warning message if the user has not set any analytics preferences
+- hides the main cookie banner component if present (to avoid confusion for the user)
+
+On submitting the form:
+- sets a `cookie_policy` cookie containing the user's analytics preferences (true or false)
+- sets a `seen_cookie_message` cookie to hide the cookie banner for the user for 365 days
```

Comment:
These will be prefixed `dm_`

Reply:
We may not actually need `seen_cookie_message` - can rely on the existence of `dm_cookie_policy`